alembic/autogenerate/render.py

```diff
@@ -721,4 +721,10 @@ def _render_check_constraint(constraint, autogen_context):
     }
 
 
+@renderers.dispatch_for(ops.ExecuteSQLOp)
+def _execute_sql(autogen_context, op):
+    assert isinstance(op.sqltext, str)
```

Comment:
can we do a more full blown "if not isinstance(op.sqltext, compat.string_types): raise NotImplementedError("Autogenerate rendering of SQL Expression language constructs not supported here; please use a plain SQL string")"  ?